addons/website_event_sale/views/website_sale_templates.xml

```diff
@@ -22,7 +22,7 @@
     <!-- If the sale order line concerns an event, we want to show an additional line with the event name even on small screens -->
     <template id="cart_lines_inherit_website_event_sale" inherit_id="website_sale.cart_lines_price" name="Event Shopping Cart Lines">
         <xpath expr="//del" position="attributes">
-            <attribute name="t-attf-class" separator=" " add="#{line.event_id and 'd-none' or ''}"/>
+            <attribute name="class" separator=" " add="#{line.event_id and 'd-none' or ''}"/>
```

Comment:
Well no you can't add that to class otherwise it won't be evaluated right?

Either we need to add back the original class attribute somehow (don't think it's possible). Or go back to using t-attf in the original template, or use t-if (unless there's a really good reason not to use it, seems most straightfoward)

Reply:
Indeed, my bad 😅 I think using a `t-if` here is the most straightforward solution, I'll update the code and check again. Thanks for your review! 
```suggestion
            <attribute name="class" separator=" " add="#{line.event_id and 'd-none' or ''}"/>
```